src/service/message.js

```diff
@@ -1,16 +1,41 @@
 const fs = require('fs');
+const readline = require('readline');
 
-const addMessage = (message) => {
+const createInterface = () => readline.createInterface({
+  input: fs.createReadStream('log.txt'),
+  output: process.stdout,
+  terminal: false
+});
+
+const addMessage = (newMessage) => new Promise((resolve, reject) => {
   let lineBreak = '';
 
-  if (fs.existsSync('log.txt') && fs.readFileSync('log.txt', 'utf8') !== '') {
-    lineBreak = '\n';
-  }
+  if (fs.existsSync('log.txt')) {
+    const rl = createInterface();
+    let newId = 0;
 
-  fs.appendFileSync('log.txt', `${lineBreak}${message}`);
+    rl.on('line', line => {
+      if (line) {
+        const { id, message } = JSON.parse(line);
 
-  return message;
-};
+        fs.appendFileSync('log1.txt', `${lineBreak}${JSON.stringify({ id, message })}`);
+
+        if (!lineBreak) { lineBreak = '\n'; }
+
+        newId = id;
+      }
+    });
+
+    rl.on('close', () => {
+      fs.unlinkSync('log.txt');
+      fs.appendFileSync('log1.txt', `${lineBreak}${JSON.stringify({ id: ++newId, message: newMessage })}`);
+      fs.renameSync('log1.txt', 'log.txt');
+      resolve(newMessage);
+    });
+  } else {
+    reject(new Error('file not exiest'));
```

Comment:
So not it's impossible to add message if message wasn't added before. But it couldn't be added before because it's impossible to add message if file doesn't exist.
We should create file if it doesn't exist.